### DISCUSSION TITLE
主函数关联了许多CPP文件下的函数，该怎么将CPP文件填进去呢？

### DISCUSSION THREAD

**Discussion by ipanda555** [Yang-Xijie/yang-xijie.github.io#15] [Cpp in VSCode on macOS]
"cpp": "g++ $fullFileName -o $dir\"$fileNameWithoutExt\"\".out\" -W -Wall -O2 -std=c++17 && $dir\"$fileNameWithoutExt\"\".out\""如果我的主函数关联了许多CPP函数，该怎么将这些CPP文件填进去呢

**Comment by Yang-Xijie** [comment]
一个一个填进去……当然了你也可以用通配符`*`一次指定多个文件
或者用cmake这类工具做成一个一个库
或者用IDE省事儿

**Comment by Yang-Xijie** [comment]
可以看一下这两个仓库  https://github.com/Yang-Xijie/Cpp-Makefile-template  https://github.com/Yang-Xijie/C-Makefile-template

`Makefile` 中的 `SOURCES = main.c # All source files of the project.` 可以直接在这里添加
